api/context.go

```diff
@@ -25,8 +25,8 @@ type ContextInterface interface {
 	Contexts(vcs, org string) (*[]Context, error)
 	ContextByName(vcs, org, name string) (*Context, error)
 	DeleteContext(contextID string) error
-	CreateContext(vcs, org, name string) (error)
-
+	CreateContext(vcs, org, name string) error
+	CreateContextWithOrgID(orgID *string, name string) error
```

Comment:
should `orgID` be `string` or `*string`?

I _think_ the main distinction between the two is that `string` is stricter in that it cannot be `nil` whereas `*string` can. With `*string` you'd have to make sure the pointer is `nil` anywhere you dereference it (I found this [quick reference](https://dhdersch.github.io/golang/2016/01/23/golang-when-to-use-string-pointers.html) insightful).